client/transactions/list/index.tsx

```diff
@@ -77,6 +78,28 @@ interface Column extends TableCardColumn {
 	cellClassName?: string;
 }
 
+const applyThousandSeparator = ( trxCount: number ) => {
+	const siteLang = document.documentElement.lang;
+	const siteNumberOptions = {
+		thousandSeparator: ',',
+	};
+
+	if ( [ 'fr', 'pl' ].some( ( lang ) => siteLang.startsWith( lang ) ) ) {
+		siteNumberOptions.thousandSeparator = ' ';
```

Comment:
I am not sure about using space and `'` as a separator tbh. 🤔 Could you start a thread and confirm this in the team channel once?

Reply:
Resolving based on Slack thread: p1660118893096929-slack-C01BPL3ALGP